### ISSUE TITLE
Unhelpful Errors: not instantiating table name parameters

### ISSUE THREAD

**Issue by combinatorist** [prestodb/presto#10836]
## Problem
While running a query, I get the following error verbatim:
```
Query 20180614_162701_00013_n2eqp failed: Table %s.%s was dropped by another transaction
```

## Cause?
It seems the error message is not instantiating the parameterized table name, here: https://github.com/prestodb/presto/blob/master/presto-hive/src/main/java/com/facebook/presto/hive/metastore/SemiTransactionalHiveMetastore.java#L473
I'm guessing this would effect other error messages I noticed in the same source code, but I don't know Java.

## Impact
This is a bummer, because my query uses 45 tables, so it leaves me to guess and check.

## Environment
```
$ presto-cli --version
Presto CLI 0.194
$ hive --version
Hive 2.3.2-amzn-2
Git git://ip-10-231-79-134/workspace/workspace/bigtop.release-rpm-5.13.0/build/hive/rpm/BUILD/apache-hive-2.3.2-amzn-2-src -r 300391394352b074b85b529e870816a72c6f314a
Compiled by ec2-user on Thu Mar 22 00:00:05 UTC 2018
From source with checksum b29ae44dc2e4fae76b6fc4a9445fedbc
```
(on AWS EMR)

Let me know if you need any other info.

**Comment by haozhun** [comment]
This was fixed 3 months ago in 2328f43e1290b0b2a5ecf3e714367c6ecf9418f0.

Release 0.198 and after have the fix.

**Comment by combinatorist** [comment]
Sorry, @haozhun, I didn't realize. Thanks for the update!
